[rospy.client][INFO] 2020-01-31 22:53:42,242: init_node, name[/sr_determine_colors_4325_1580491422206], pid[4325]
[xmlrpc][INFO] 2020-01-31 22:53:42,243: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-01-31 22:53:42,243: Started XML-RPC server [http://itachi-HP-Pavilion-Laptop-15-cc1xx:44119/]
[rospy.init][INFO] 2020-01-31 22:53:42,243: ROS Slave URI: [http://itachi-HP-Pavilion-Laptop-15-cc1xx:44119/]
[rospy.impl.masterslave][INFO] 2020-01-31 22:53:42,243: _ready: http://itachi-HP-Pavilion-Laptop-15-cc1xx:44119/
[rospy.registration][INFO] 2020-01-31 22:53:42,244: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-01-31 22:53:42,244: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-01-31 22:53:42,344: registered with master
[rospy.rosout][INFO] 2020-01-31 22:53:42,344: initializing /rosout core topic
[rospy.rosout][INFO] 2020-01-31 22:53:42,347: connected to core topic /rosout
[rospy.simtime][INFO] 2020-01-31 22:53:42,349: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-01-31 22:53:42,359: topic[/usb_cam/image_rect_color] adding connection to [http://itachi-HP-Pavilion-Laptop-15-cc1xx:43669/], count 0
[rosout][ERROR] 2020-01-31 22:53:42,548: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7f4022fe47a0>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector.py", line 71, in callback
    for i in self.rect_list.keys():
AttributeError: sr_determine_colors instance has no attribute 'rect_list'

[rospy.internal][INFO] 2020-01-31 22:53:42,561: topic[/rosout] adding connection to [/rosout], count 0
[rosout][ERROR] 2020-01-31 22:53:42,704: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7f4022fe47a0>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector.py", line 71, in callback
    for i in self.rect_list.keys():
AttributeError: sr_determine_colors instance has no attribute 'rect_list'

[rosout][ERROR] 2020-01-31 22:53:42,852: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7f4022fe47a0>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector.py", line 71, in callback
    for i in self.rect_list.keys():
AttributeError: sr_determine_colors instance has no attribute 'rect_list'

[rosout][ERROR] 2020-01-31 22:53:43,033: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7f4022fe47a0>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector.py", line 71, in callback
    for i in self.rect_list.keys():
AttributeError: sr_determine_colors instance has no attribute 'rect_list'

[rosout][ERROR] 2020-01-31 22:53:43,179: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7f4022fe47a0>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector.py", line 71, in callback
    for i in self.rect_list.keys():
AttributeError: sr_determine_colors instance has no attribute 'rect_list'

[rosout][ERROR] 2020-01-31 22:53:43,337: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7f4022fe47a0>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector.py", line 71, in callback
    for i in self.rect_list.keys():
AttributeError: sr_determine_colors instance has no attribute 'rect_list'

[rosout][ERROR] 2020-01-31 22:53:43,495: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7f4022fe47a0>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector.py", line 71, in callback
    for i in self.rect_list.keys():
AttributeError: sr_determine_colors instance has no attribute 'rect_list'

[rosout][ERROR] 2020-01-31 22:53:43,648: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7f4022fe47a0>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector.py", line 71, in callback
    for i in self.rect_list.keys():
AttributeError: sr_determine_colors instance has no attribute 'rect_list'

[rosout][ERROR] 2020-01-31 22:53:43,808: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7f4022fe47a0>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector.py", line 71, in callback
    for i in self.rect_list.keys():
AttributeError: sr_determine_colors instance has no attribute 'rect_list'

[rosout][ERROR] 2020-01-31 22:53:43,966: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7f4022fe47a0>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector.py", line 71, in callback
    for i in self.rect_list.keys():
AttributeError: sr_determine_colors instance has no attribute 'rect_list'

[rosout][ERROR] 2020-01-31 22:53:44,165: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7f4022fe47a0>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector.py", line 71, in callback
    for i in self.rect_list.keys():
AttributeError: sr_determine_colors instance has no attribute 'rect_list'

[rosout][ERROR] 2020-01-31 22:53:44,322: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7f4022fe47a0>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector.py", line 71, in callback
    for i in self.rect_list.keys():
AttributeError: sr_determine_colors instance has no attribute 'rect_list'

[rosout][ERROR] 2020-01-31 22:53:44,471: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7f4022fe47a0>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector.py", line 71, in callback
    for i in self.rect_list.keys():
AttributeError: sr_determine_colors instance has no attribute 'rect_list'

[rosout][ERROR] 2020-01-31 22:53:44,635: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7f4022fe47a0>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector.py", line 71, in callback
    for i in self.rect_list.keys():
AttributeError: sr_determine_colors instance has no attribute 'rect_list'

[rosout][ERROR] 2020-01-31 22:53:44,799: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7f4022fe47a0>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector.py", line 71, in callback
    for i in self.rect_list.keys():
AttributeError: sr_determine_colors instance has no attribute 'rect_list'

[rosout][ERROR] 2020-01-31 22:53:44,947: bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7f4022fe47a0>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector.py", line 71, in callback
    for i in self.rect_list.keys():
AttributeError: sr_determine_colors instance has no attribute 'rect_list'

[rospy.core][INFO] 2020-01-31 22:53:45,095: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-01-31 22:53:45,098: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-01-31 22:53:45,098: topic[/usb_cam/image_rect_color] removing connection to http://itachi-HP-Pavilion-Laptop-15-cc1xx:43669/
[rospy.impl.masterslave][INFO] 2020-01-31 22:53:45,098: signal-2
[rospy.topics][WARNING] 2020-01-31 22:53:45,116: during shutdown, bad callback: <bound method sr_determine_colors.callback of <__main__.sr_determine_colors instance at 0x7f4022fe47a0>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/itachi/catkin_ws/src/survey_and_rescue/scripts/beacon_detector.py", line 71, in callback
    for i in self.rect_list.keys():
AttributeError: sr_determine_colors instance has no attribute 'rect_list'

[rospy.core][INFO] 2020-01-31 22:53:45,230: signal_shutdown [atexit]
